File: C++/bee 1052.cpp
#include <iostream>
#include <iomanip>
#include <cmath>

using namespace std;

int main() {

    string meses[] = {"January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"};

    int n;
    cin >> n;

    cout << meses[n-1] << endl;


    return 0;
}

File: Rust/bee_1052.rs
use std::io::{self, Read};

fn main() {
    let mut input = String::new();
    io::stdin().read_to_string(&mut input).unwrap();

    let valor: i32 = input.trim().parse().unwrap();

    let mes: &str = match valor {
        1 => "January",
        2 => "February",
        3 => "March",
        4 => "April",
        5 => "May",
        6 => "June",
        7 => "July",
        8 => "August",
        9 => "September",
        10 => "October",
        11 => "November",
        12 => "December",
        _ => "",
    };
    
    println!("{}", mes);
}
